tests/util/test_marginal.py

```diff
@@ -0,0 +1,103 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import numpy as np
+import tensorflow as tf
+
+from edward.models import Normal, InverseGamma
+from tensorflow.contrib.distributions import bijectors
```

Comment:
I found this test very intuitive. Great work. One note: you don't use the `bijectors` module